tests/plugins/pastefromword/manual/pastefromword.md

```diff
@@ -0,0 +1,11 @@
+@bender-tags: 4.15.0, word, feature, generic
```

Comment:
It would be easier to verify if content was pasted correctly if the editor window was bigger. Let's set editor's height to `500` maybe?